### ISSUE TITLE
Possible incomplete lasttouched log on canisterbombs

### ISSUE THREAD

**Issue by ghost** [vgstation-coders/vgstation13#385]
[bug]
Had a pull on a canisterbomb in a cult round, because fukken cult mang, on the shuttle with us on the way to centcomm. Some chucklefuck thought they'd be cute and set it off. Admin response showed me as lasttouched when it wasn't me that set it off. Presumably because of the Pull?
![possible canisterbomb log error](https://f.cloud.github.com/assets/5564796/1881922/1e10ba18-796e-11e3-9c6d-cc2073b185b0.png)

Possible steps to reproduce?
Have one mob with a primed canisterbomb with pull active
Have another mob open the valve rending everything to ded

Add a LastOpenedValve var for log?
Maybe a opened valve adminlog message if it creates an explosion?

Forgot the guide sorry chaps. In general:

> > 57374662
> > Get a Canister of hotmix. plasma lets you use more plasma/oxygen in the tank. I generally go
> > <=4000C CO2 - 150 stock plasma 300 stock O2
> > 4000C CO2 - 200 stock plasma 400 stock O2
> > 7000C CO2 or >6000C Plasma- 300 stock plasma 600 stock O2
> > Put the tank in the canister, set target pressure to max, make sure your bombsuit is on open the valve and let the explosions caress you

tl;dr:
- Roundstart
- Get tools+multitool and hack the air alarm in toxins and pulse wires until you find the ID lock. Turn off vent #2 and if the scubber is siphoninbg either hop over to scrubber to fix or mash activate panic syphon twice.
- mix a can 1 plasma 2 oxy on the 3-connector hookkup
- Open the inner airlock on the burnchamber. Set the fuelmix flow in to 4500kPa. Set the gas extractor to 4500kPa
- Close the airlock door
- Drag a fuel canister down and wrench to connect. Open the manual valve so it can start to fill the burnchamber
- When the plasma sprite shows in the burnchamber hit the ignite button
- Drag a canister of room temp plasma and room temp oxygen if you haven't already into toxins
- Put an oxygen tank in the portable air pump, set target pressure to max, and turn it on. After about 2-3 seconds turn it off
- Keep an eye on your fuelmix canister. Once the red light is blinking it's at <10kPa and you need to swap in a second
- Once the second is empty let it burn for about 30-60 seconds then re-hack the air alarm and turn vent #2 back on
- Wrench and empty canister to the connector and open the manual valve
- Since you're probably using a CO2 hotmix setup instead of a double thermal plate plasma hookup use 150-200 stock plasma and 300-400 stock oxygen in the emptied oxygen tank.  Since your CO2 is at 8000C both should work anyway but if you ever turn the vavle and the tank neither blows up or ruptures without exploding you need to reduce your contents so it can get hotter or more shit that can burn
- Set target pressure on the canister bomb to max
- Drag to testing area
- Put tank in to prime it
